.gitignore

```diff
@@ -1,2 +1,3 @@
 /target
 openocd.gdb
+Embed.toml
```

Comment:
```suggestion
```

Neither should be ignored. Both should be tracked. If people need custom scripts then we need to find another way.